### PR TITLE
Upload database script

### PR DESCRIPTION
**Pull request by h-mayorquin**
To discuss and divide the computational load.

### CODE REVIEW

File: python/upload_templates.py [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+]
+dandiset_paths_with_ecephys.sort()
+dandiset_paths_with_ecephys = [
+    path for path in dandiset_paths_with_ecephys if "KS" in path
+]
+
+
+for asset_path in dandiset_paths_with_ecephys:
+
+    asset_path = dandiset_paths_with_ecephys[3]
```
Comment (h-mayorquin) [inline python/upload_templates.py]:
```suggestion
```

File: python/upload_templates.py [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+    recording_asset = dandiset.get_asset_by_path(path=asset_path)
+    url = recording_asset.get_content_url(follow_redirects=True, strip_query=True)
+    file_path = url
+
+    electrical_series_path = "acquisition/ElectricalSeriesAp00"
```
Comment (h-mayorquin) [inline python/upload_templates.py:43]: @alejoe91 I think this the 00 at the end of the path is restricting us to only one the two mounted probes? What do you think?

File: python/upload_templates.py [hunk 3]
```diff
@@ -0,0 +1,207 @@
...
+            sorting_pid = pid
+            break
+
+    sorting = IblSortingExtractor(
+        pid=sorting_pid, one=one_instance, good_clusters_only=True
+    )
+
+
+
+    ibl_recording = IblRecordingExtractor(pid=pid, stream_name=f"probe{probe_number}.ap" )
```
Comment (h-mayorquin) [inline python/upload_templates.py:73]: @alejoe91 this works and gets me what I expect but can you double check this?

File: python/upload_templates.py [hunk 4]
```diff
@@ -0,0 +1,207 @@
...
+    )
+
+    random_spike_parameters = {
+        "method": "all",
+    }
+
+    template_extension_parameters = {
+        "ms_before": 3.0,
+        "ms_after": 5.0,
+        "operators": ["average", "std"],
```
Comment (h-mayorquin) [inline python/upload_templates.py]:
```suggestion
        "operators": ["average"],
```
I guess we don't need the std here, right?

**Review by h-mayorquin — COMMENTED** [review]
For us to discuss, @alejoe91 .

Two big points:
1) Sometimes this is failing without https://github.com/SpikeInterface/spikeinterface/pull/2796. Sam seems OK with it, maybe we can ask him again and merge?
2) I am wondering if now that are traversing the whole data with the analyzer we can also make an estimation of the noise.

File: python/upload_templates.py (hunk 2 above)
Comment (alejoe91) [reply python/upload_templates.py:43]: Yes, maybe we can use the `find_electrical_series` and add different AP streams as different datasets

File: python/upload_templates.py (hunk 4 above)
Comment (alejoe91) [reply python/upload_templates.py]: right

File: python/upload_templates.py (hunk 1 above)
Comment (alejoe91) [reply python/upload_templates.py]: so this should be removed?

File: python/upload_templates.py [hunk 5]
```diff
@@ -0,0 +1,207 @@
...
+    )
+
+    pre_processed_recording = common_reference(
+        highpass_filter(
+            phase_shift(astype(recording=recording_cache, dtype="float32")),
+            freq_min=1.0,
+        )
+    )
+
+    # take first and last minute
```
Comment (alejoe91) [inline python/upload_templates.py]:
```suggestion
```
This comment threw me off :)

File: python/upload_templates.py (hunk 2 above)
Comment (h-mayorquin) [reply python/upload_templates.py:43]: Let me test this.

File: python/upload_templates.py (hunk 1 above)
Comment (h-mayorquin) [reply python/upload_templates.py]: Yes. Done.

File: python/upload_templates.py (hunk 4 above)
Comment (h-mayorquin) [reply python/upload_templates.py]: OK.

File: python/upload_templates.py (hunk 5 above)
Comment (h-mayorquin) [reply python/upload_templates.py]: Indeed. Removed.

**Review by alejoe91 — APPROVED** [review]

**Pull request merged**